CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+# Contributing to buidler-aragon
+
+:tada: Thank you for being interested in contributing to an Aragon Project! :tada:
+
+Feel welcome and read the following sections in order to know how to ask questions and how to work on something.
+
+All members of our community are expected to follow our [Code of Conduct](https://wiki.aragon.org/documentation/Code_of_Conduct/). Please make sure you are welcoming and friendly in all of our spaces.
+
+## Get Rewarded
```

Comment:
I'd probably just leave this section out for now.

Reply:
Fixed with https://github.com/aragon/buidler-aragon/pull/132/commits/36cdee616e35e5ce507b29c6a5511e6807737474